syntax/puppet.vim

```diff
@@ -1,167 +1,433 @@
-" puppet syntax file
-" Filename:     puppet.vim
-" Language:     puppet configuration file
-" Maintainer:   Luke Kanies <luke@madstop.com>
-" URL:
-" Last Change:
-" Version:
```

Comment:
There's only one other file in this repo that uses a 'Last change'. It's the indent/puppet.vim that contains:

" Last Change: 2009 Aug 19

which is obviously quite incorrect. I can add it, but are you sure you want this field added with the burden of maintaining it? Doesn't seem to be something that anyone cared much about in the past.